fastapi_auth_api_key/tests/test_fastapi_api_key_dependencies.py

```diff
@@ -72,34 +72,36 @@ def setUpClassApiKey(cls):
     def test_authenticated_auth_api_key_no_api_key(self):
         # An exception is raised when no api key is used
         with self.assertRaises(HTTPException) as error:
-            authenticated_auth_api_key(False, self.demo_env, self.demo_endpoint)
-        self.assertEqual(error.exception.detail, "Missing HTTP-API-KEY header")
+            authenticated_auth_api_key(self.demo_env, self.demo_endpoint)
+        self.assertEqual(error.exception.detail, "Missing HTTP API key header")
 
     def test_authenticated_auth_api_key_no_api_key_found(self):
+        self.demo_endpoint.auth_http_api_key_header = "404"
```

Comment:
These testes are wrong

You should not change the `auth_http_api_key_header` field.
That's supposed to be the HTTP header and it's always the same.

You should change the value and ensure it actually authenticates